{"level":"error","message":"Test error","service":"my-project"}
{"level":"error","message":"Test error","service":"my-project"}
{"level":"error","message":"Test error","service":"my-project"}
{"level":"error","message":"Server threw error test error","service":"my-project","stack":"Error: test error\n    at Server.<anonymous> (/home/tudor/Documents/Articles/NodeJS TS API/my-project/src/server.ts:14:23)\n    at Object.onceWrapper (node:events:509:28)\n    at Server.emit (node:events:390:28)\n    at Server.emit (node:domain:475:12)\n    at emitListeningNT (node:net:1368:10)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Server threw error test error","service":"my-project","stack":"Error: test error\n    at Server.<anonymous> (/home/tudor/Documents/Articles/NodeJS TS API/my-project/src/server.ts:14:23)\n    at Object.onceWrapper (node:events:509:28)\n    at Server.emit (node:events:390:28)\n    at Server.emit (node:domain:475:12)\n    at emitListeningNT (node:net:1368:10)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Server threw error test error","service":"my-project","stack":"Error: test error\n    at Server.<anonymous> (/home/tudor/Documents/Articles/NodeJS TS API/my-project/src/server.ts:14:23)\n    at Object.onceWrapper (node:events:509:28)\n    at Server.emit (node:events:390:28)\n    at Server.emit (node:domain:475:12)\n    at emitListeningNT (node:net:1368:10)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Server threw error test error","service":"my-project","stack":"Error: test error\n    at Server.<anonymous> (/home/tudor/Documents/Articles/NodeJS TS API/my-project/src/server.ts:14:23)\n    at Object.onceWrapper (node:events:509:28)\n    at Server.emit (node:events:390:28)\n    at Server.emit (node:domain:475:12)\n    at emitListeningNT (node:net:1368:10)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Server threw error test error","service":"my-project","stack":"Error: test error\n    at Server.<anonymous> (/home/tudor/Documents/Articles/NodeJS TS API/my-project/src/server.ts:14:23)\n    at Object.onceWrapper (node:events:509:28)\n    at Server.emit (node:events:390:28)\n    at Server.emit (node:domain:475:12)\n    at emitListeningNT (node:net:1368:10)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Server threw error test error","service":"my-project","stack":"Error: test error\n    at Server.<anonymous> (/home/tudor/Documents/Articles/NodeJS TS API/my-project/src/server.ts:14:23)\n    at Object.onceWrapper (node:events:509:28)\n    at Server.emit (node:events:390:28)\n    at Server.emit (node:domain:475:12)\n    at emitListeningNT (node:net:1368:10)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Server threw error test error","service":"my-project","stack":"Error: test error\n    at Server.<anonymous> (/home/tudor/Documents/Articles/NodeJS TS API/my-project/src/server.ts:14:23)\n    at Object.onceWrapper (node:events:509:28)\n    at Server.emit (node:events:390:28)\n    at Server.emit (node:domain:475:12)\n    at emitListeningNT (node:net:1368:10)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)"}
